Integrations/AWS-ACM/AWS-ACM.py

```diff
@@ -0,0 +1,355 @@
+        kwargs.update({'CertificateStatuses': args.get('certificateStatuses')})
+    if args.get('extendedKeyUsage') is not None:
+        includes.update({'extendedKeyUsage': [args.get('extendedKeyUsage')]})
+    if args.get('keyUsage') is not None:
+        includes.update({'keyUsage': [args.get('keyUsage')]})
+    if args.get('keyTypes') is not None:
+        includes.update({'keyTypes': [args.get('keyTypes')]})
+    if includes:
+        kwargs.update({'Includes': includes})
+
+    response = client.list_certificates(**kwargs)
+    for cert in response['CertificateSummaryList']:
+        data.append({
+            'CertificateArn': cert['CertificateArn'],
+            'DomainName': cert['DomainName'],
+            'Region': obj['_user_provided_options']['region_name'],
+        })
+
+    ec = {'AWS.ACM.Certificates(val.CertificateArn === obj.CertificateArn)': data}
+    human_readable = tableToMarkdown('AWS ACM Certificates', data)
+    return_outputs(human_readable, ec)
+
+
+def add_tags_to_certificate(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+    kwargs = {
+        'CertificateArn': args.get('certificateArn'),
+        'Tags': parse_tag_field(args.get('tags'))
+    }
+    response = client.add_tags_to_certificate(**kwargs)
+    if response['ResponseMetadata']['HTTPStatusCode'] == 200:
+        demisto.results("The Certificate was Tagged successfully")
+
+
+def remove_tags_from_certificate(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+    kwargs = {
+        'CertificateArn': args.get('certificateArn'),
+        'Tags': parse_tag_field(args.get('tags'))
+    }
+    response = client.remove_tags_from_certificate(**kwargs)
+    if response['ResponseMetadata']['HTTPStatusCode'] == 200:
+        demisto.results("The Certificate Tags were removed successfully")
+
+
+def list_tags_for_certificate(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+
+    kwargs = {'CertificateArn': args.get('certificateArn')}
+    response = client.list_tags_for_certificate(**kwargs)
+
+    data = ({'CertificateArn': args.get('certificateArn')})
+    for tag in response['Tags']:
+        data.update({
+            tag['Key']: tag['Value']
+        })
+
+    ec = {'AWS.ACM.Certificates(val.CertificateArn === obj.CertificateArn).Tags': data}
+    human_readable = tableToMarkdown('AWS ACM Certificate Tags', data)
+    return_outputs(human_readable, ec)
+
+
+def get_certificate(args):
```

Comment:
please return the certificate and the certificate chain in 2 separate files and do not print them in an entry or a log.